cmd/ttn-lw-cli/commands/end_devices_split.go

```diff
@@ -339,3 +340,46 @@ func deleteEndDevice(ctx context.Context, devID *ttnpb.EndDeviceIdentifiers) err
 
 	return nil
 }
+
+func updateDeviceLocation(device *ttnpb.EndDevice, flags *pflag.FlagSet) bool {
+	latitude, _ := flags.GetFloat64("location.latitude")
+	longitude, _ := flags.GetFloat64("location.longitude")
+	altitude, _ := flags.GetInt32("location.altitude")
+	accuracy, _ := flags.GetInt32("location.accuracy")
+
+	if latitude == 0 && longitude == 0 && altitude == 0 && accuracy == 0 {
+		return false
+	}
+	// need both latitude and longitude
```

Comment:
Why's that? Can't we have devices on the equator or the prime meridian?

Reply:
Using `flags.Changed` in order to figure out whether the flag was actually passed.